editor/src/document/document_file.rs

```diff
@@ -359,7 +369,10 @@ impl DocumentMessageHandler {
 		}
 	}
 
-	pub fn redo(&mut self) -> Result<(), EditorError> {
+	pub fn redo(&mut self, responses: &mut VecDeque<Message>) -> Result<(), EditorError> {
+		// Push the GetOpenDocumentsList message to the bus in order to update the save status of the open documents
```

Comment:
The GetOpenDocumentsList Message sholud probably be renamed to UpdateOpenDocumentsList, righ?

Reply:
changed